pages/api/programData/_middleware.js

```diff
@@ -15,3 +17,46 @@ export async function middleware(req, ev) {
     }
   }
 }
+
+export async function GetProgramData(req, res, getPath, getMockObj, mapCard) {
```

Comment:
This logic belongs in /lib

Reply:
thanks for the tip - I was also thinking similarly